src/test01/app.js

```diff
@@ -1,8 +1,16 @@
+import './app.scss';
+
 export const test01 = {
-    template: require('./app.html'),
-    controller() {
+	template: require('./app.html'),
+	controller($scope, $http, buttonService) {
+		$scope.buttons = [];
 
-        // your code here
+		function initialize() {
+			buttonService.getData().then(function (response) {
+				$scope.buttons = response.data;
+			});
+		}
 
-    }
+		initialize();
+	}
 };
```

Comment:
This controller calls buttonService to retrieve the JSON object that contains the text for each of the buttons. The response is then stored in the buttons object in the controller's scope.